### DISCUSSION TITLE
Проблема z-index у yandex-map-marker

### DISCUSSION THREAD

**Discussion by alekseyseysey** [yandex-maps-unofficial/vue-yandex-maps#347] [Q&A (2.x)]
Добрый день! Подскажите пожалуйста, сначала делал через yandex-map-default-marker, была проблема что маркеры заплывали на popup, нашел ответ на этот вопрос и по вашему совету переделал под yandex-map-marker. Сначала задал z-index в settings для маркера, но он задается для всех маркеров и получается что в этом нет смысла. Попытался отслеживать маркеры по отдельности, но и тут провал, z-index задается, но потом все равно возвращается в исходное положение, то есть '0'. Что делать в таких случаях?

**Comment by daniluk4000** [comment]
Добрый день. Вы когда попап открываете, кидаете туда ивент открытия вроде как, вот когда у маркера есть этот ивент и ставьте ему больше zIndex. Сбрасываться он, конечно, не должен.

**Comment by alekseyseysey** [comment]
Спасибо, буду пробовать!
